core/src/utils/file/file_utils.py

```diff
@@ -94,3 +94,7 @@ def get_output_path(input_path: Union[str, Path], output_suffix: str) -> Path:
     output_filename = get_output_filename(input_path, output_suffix)
 
     return parent_dir / output_filename
+
+
+def find_files_by_regex(path: Path, regex: re.Pattern) -> List[Path]:
```

Comment:
I think the return type is correct

Reply:
I mean we should change it to `List[str]`, because `os.listdir` returns a list of strings.

<img width="692" alt="image" src="https://github.com/hyperskill/hyperstyle-analysis-prod/assets/55441714/03d60d2a-a04e-42fa-b22a-e3f5fbb98864">